OfflineService/OfflineDataRegionTableViewCell.swift

```diff
@@ -56,43 +56,63 @@ class OfflineDataRegionTableViewCell: UITableViewCell {
     }
     
     // MARK: - Public UI update related methods
-    
-    func showDownloadProgress(for domain: OfflineDataDomain, dataPack: OfflineDataPack, metadata: OfflineDataRegionMetadata) {
+
+    func showDownloadProgress(for domain: OfflineRegionDomain, dataPack: OfflineRegionPack, region: OfflineRegion) {
         DispatchQueue.main.async {
```

Comment:
I think after changes which were introduced in `OfflineService` `onDownloading(for:metadata:pack:)` is already calling `didStartDownloading(region:forDomain:)` on main thread, so there is no need to perform double dispatch.